subsys/config/Kconfig

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig CONFIG_SYS_ENABLED
+	bool
+	default n
+	prompt "Enable nonvolatile configuration system"
+	depends on FILE_SYSTEM_NFFS || FCB
```

Comment:
does this really depend on NFFS, or is it file systems in general?

Reply:
FS part of the configuration system relay on pure FS API. The Reason behind limit it to NFFS is lack of files-rename implementation in fatfs code.